docs/frontend.md

```diff
@@ -1 +1,51 @@
 # Frontend
+This site is built using Drupal and Federalist with a USWDS theme.
+
+## Theme commands
+These commands should be run inside web/themes/custom/vote_gov
+| **Command**      | **Use case**                                                                      |
+|------------------|-----------------------------------------------------------------------------------|
+| `npm run build`  | Build assets with gulp                                                            |
+| `npm run dev`    | Run scss linter                                                                   |
+
+## Theme file structure
+```
+web/
+├── core
+├── modules
+├── profiles
+├── sites
+└── themes/
+    └── custom/
+        └── vote_gov/
+            ├── ckeditor
+            ├── config
```

Comment:
We should delete `config` this is only needed for setting config on first installation.